rest_framework_json_api/utils.py

```diff
@@ -286,23 +284,6 @@ def get_default_included_resources_from_serializer(serializer):
     return list(getattr(meta, "included_resources", []))
 
 
-def get_included_serializers(serializer):
```

Comment:
This is public API. I am in favor of deleting this but this has to go through deprecation first. So best is when this method returns `serializer.included_serializers` and writes a deprecation warning to use `serializer.included_serializers` instead.